docs/CONFIG-VARS.md

```diff
@@ -72,6 +74,40 @@ You can use `default_public_access_cidrs` to set a default range for all created
 | vm_public_access_cidrs | IP Ranges allowed to access the VMs | list of strings | | opens port 22 for SSH access to the jump and/or nfs VM |
 | postgres_access_cidrs | IP Ranges allowed to access the AWS PostgreSQL Server | list of strings |||
 
+## Networking
+ | Name | Description | Type | Default | Notes |
+ | :--- | ---: | ---: | ---: | ---: |
+ | vpc_cidr | Address space for the VPC | string | "192.168.0.0/16" | This variable is ignored when `vpc_id` is set (aka bring your own VPC) |
+ | subnets | Subnets to be created and their settings | map | check below | This variable is ignored when subnet_ids is set (aka bring your own subnets). All defined subnets must exist within the vnet address space. |
+
+The default values for the subnets variable are:
+
+```yaml
+{
+  "private" : ["192.168.0.0/18", "192.168.64.0/18"],
+  "public" : ["192.168.129.0/25", "192.168.129.128/25"],
+  "database" : ["192.168.128.0/25", "192.168.128.128/25"]
+}
+```
+
+### Use Existing
+When desiring to deploy into existing vpc or subnets the variables below can be used to define the exiting resources
```

Comment:
The GCP BYO doc has a paragraph of information about BYO. It seems like we don't have enough here to identify what a customer needs to do. I'm including the GCP text here to illustrate the level of detail that I found helpful when testing GCP BYO

If desired, you can deploy into an existing VPC, use existing subnets, and provide an existing Cloud NAT IP address. You will need private subnet for the GKE nodes and a public subnet for the Jump VM and (if used) the NFS VM. The GKE subnet requires two secondary CIDR ranges for the Kubernetes Pods and Services (see https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips#cluster_sizing).

The existing subnets need match the same region given in the location variable defined here

Reply:
See latest doc changes